modules/virt-creating-offline-vm-snapshot-cli.adoc

```diff
@@ -1,11 +1,22 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
 
-[id="virt-creating-offline-vm-snapshot-cli_{context}"]
-= Creating an offline virtual machine snapshot in the CLI
+[id="virt-creating-vm-snapshot-cli_{context}"]
+= Creating an virtual machine snapshot in the CLI
 
-You can create a virtual machine (VM) snapshot for an offline VM by creating a `VirtualMachineSnapshot` object.
+You can create a virtual machine (VM) snapshot for a VM by creating a `VirtualMachineSnapshot` object.
+
+You can create a virtual machine (VM) snapshot for an online VM by creating a `XX` object. Kubevirt will coordinate with the QEMU guest agent to create a snapshot of the online VM.
+
+[IMPORTANT]
+====
+In order to create snapshots of an online (Running state) VM, you must install the QEMU guest agent.
```

Comment:
I thought we discussed softening this statement.  We recommend using the guest agent for a more reliable snapshot but it is not required.

Reply:
Yep, same as above. This file has been replaced by virt-creating-vm-snapshot-cli. Here's the softer, new wording:

[NOTE]
====
In order to create snapshots of an online (Running state) VM with the highest integrity, we recommend you install the QEMU guest agent.

The QEMU guest agent ensures a consistent snapshot by attempting to quiesce the VM's filesystem. This insures that in-flight I/O is written to the disk before the snapshot is taken and ensures that VMs with multiple disks have a coherent snapshot. If the guest agent is not present, quiescing is not possible and a best effort snapshot is taken. The conditions under which the snapshot was taken are reflected in the snapshot indications that are displayed in the web console or CLI.
====